### DISCUSSION TITLE
Mosh protocol + secure crt import profiles

### DISCUSSION THREAD

**Discussion by doreks** [Eugeny/tabby#5998] [Q&A] [answered]
I'm planning to move to this app and I have, for today just two question :

1/ Is there any plan to support mosh protocol in following release??

2/ from securecrt I can export all my sessions connections but is there any way to import them to this app?

Thanks

**Comment by Eugeny** [comment, marked as answer]
* Yes, there's plan to add mosh support
* If you can export hosts into your OpenSSH config (~/.ssh/config), Tabby will show them

**Reply by doreks** [reply]
Thanks!! More than welcome!!

**Comment by lawweiliang** [comment]
@doreks 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)

**Comment by borekon** [comment]
is it already supported?
